src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -0,0 +1,1008 @@
+/*
```

Comment:
Why all this file?

Reply:
To mark all blocks as remote. Will probably be removed in the future.
I've referred to this in SparkDPU doc